include/circt/Dialect/SV/SVVerification.td

```diff
@@ -159,3 +159,58 @@ def CoverConcurrentOp : ConcurrentVerifOp<"cover.concurrent",
     section 16.5 of the SystemVerilog 2017 specification.
   }];
 }
+
+
+/// Concurrent verification operations defined by the SV standard.
+class PropertyVerifOp<string mnemonic, list<Trait> traits = [AttrSizedOperandSegments]> :
+    SVOp<mnemonic, traits> {
+  let arguments = (ins
+    I1:$property,
+    OptionalAttr<EventControlAttr>:$event, Optional<I1>:$clock, 
+    Optional<I1>:$disable,
+    OptionalAttr<StrAttr>:$label);
+
+  let assemblyFormat = [{
+    $property (`on` $event^)? ($clock^)? (`disable_iff` $disable^)?
+    (`label` $label^)? attr-dict
+  }];
+
+  let builders = [
+    // no clock or event
+    OpBuilder<(ins "mlir::Value":$property,
+      "mlir::Value":$disable,
+      StrAttr.storageType:$label),
```

Comment:
Just curious, what's going on here?

Reply:
just some custom builders to make it easier to use the optional parameters in any order.